.gitignore

```diff
@@ -9,4 +9,6 @@
 *output_nsys.sqlite
 model-formulation/catboost_info/*
 print_outs.txt
+src/*_pb2*.py
 *__pycache__*
+*.zip
```

Comment:
What zip file is created?

Reply:
While install grpc it downloads some zip

Reply:
from https://github.com/aravindhank11/llama2-expr/pull/14/files#diff-52ed8b9ee32a55b12592adf96a1f1fb0eb7efc80831eedb3132677e53497aff7R39

This was moved from a separate install file to the common one

Reply:
Ah yes, thanks